src/metainfo/yast2-tune.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Dominique Leuenberger <dimstar@opensuse.or> -->
+<component type="addon">
+<id>yast2-tune</id>
+<extends>yast2.desktop</extends>
+<_name>Hardware Information</_name>
```

Comment:
Why `_name` and `_summary`? According to the documentation (http://www.freedesktop.org/software/appstream/docs/sect-Metadata-Addon.html) there should be `name` and `summary`.